R/modelling.R

```diff
@@ -239,11 +228,12 @@ get_exposure_matrix <- function(serodata) {
 #' serodata <- prepare_serodata(chagas2012)
 #' seromodel_object <- fit_seromodel(serodata = serodata,
 #'                                   foi_model = "constant")
-#' foi_central_estimates <- get_foi_central_estimates(seromodel_object)
+#' foi_central_estimates <- get_foi_central_estimates(seromodel_object = seromodel_object, 
+#'                                                    serodata = serodata)
 #' @export
-get_foi_central_estimates <- function(seromodel_object) {
+get_foi_central_estimates <- function(seromodel_object, serodata) {
```

Comment:
Similar as before, it seems that we need `exposure_years` rather than `serodata`. Should we change this?

Reply:
Same as above.